### ISSUE TITLE
403 error?

### ISSUE THREAD

**Issue by fonewoda** [yowmamasita/savant#1]
The link parse works.
But,when input the link,got 403 error.
I think google server will detect browser's ip...

Please check it.Thanks.

**Comment by yowmamasita** [comment]
@fonewoda yes the generated link will attach an IP so only that IP can use that link. I am still trying to figure out how to generate links without re-scraping the page.
